cypress/integration/login.js

```diff
@@ -17,22 +9,45 @@ describe('Sourcebooks login', function() {
     ];
 
     function makeid(length) {

```

Comment:
According to coding convention we should use camel case, so it should be "makeId"

Reply:
Also length parameter is not used